flow-demo/pom.xml

```diff
@@ -6,19 +6,19 @@
     <groupId>com.example.application</groupId>
     <artifactId>demo</artifactId>
     <version>3.0-SNAPSHOT</version>
-    <packaging>jar</packaging>
+    <packaging>war</packaging>
 
     <properties>
         <java.version>8</java.version>
+        <flow.version>6.0.9</flow.version>
         <vaadin.version>19.0.8</vaadin.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Rebase with flow-port branch so as we use v23 and jdk-11

Reply:
Done.